features/Support/MailPitClient.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Support;
+
+use CultuurNet\UDB3\Json;
+use GuzzleHttp\Client;
+use GuzzleHttp\RequestOptions;
+use Psr\Http\Message\ResponseInterface;
+
+final class MailPitClient
```

Comment:
I would create an interface `MailClient` and have `MailPitClient` as the concrete implementation